bytecomp/bytelink.ml

```diff
@@ -695,7 +698,9 @@ let report_error ppf = function
       fprintf ppf "Error on dynamically loaded library: %a"
         Location.print_filename file
   | Required_module_unavailable s ->
-      fprintf ppf "Required module `%s' is unavailable" s
+      fprintf ppf "Required module '%s' is unavailable" s
```

Comment:
Why the elimination of the back-tick here? FWIW, I'd prefer to see other instances of `'%s'` changed to `` `%s'`` rather than this way around?

Reply:
@dra27 some people from your faculty [may disagree](https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html) ;-)

Reply:
Quite possibly - but let's remove them wholesale, rather than piecemeal with other changes 🙂

Reply:
(I'd also prefer them either to be post-processed to the correct Unicode character before going to the console, or for us finally to allow the correct Unicode characters directly in the message, both of which would keep Markus happy!)